### PR TITLE
Increase the log level to warning from debug

### PR DESCRIPTION
**Pull request by Julie-CAO**
libvirtd.log with debug level is over 100G on XEN host which use up the disk space ~ we have to record warning level output for now to continue the tests then investigate this issue.

- Failed tests in OSD: https://openqa.nue.suse.com/tests/4694347
- Verification run: https://openqa.nue.suse.com/tests/4701457

### CODE REVIEW

**Review by foursixnine — APPROVED** [review]
Can you squash the commits?

**Comment by Julie-CAO** [comment]
> Can you squash the commits?

Yes, thank you.

**Comment by Julie-CAO** [comment]
@alice-suse  @guoxuguang  @waynechen55 Please help review.

File: tests/virt_autotest/virt_utils.pm [hunk 1]
```diff
@@ -34,8 +34,8 @@ sub enable_debug_logging {
     # turn on debug for libvirtd
     my $libvirtd_conf_file = "/etc/libvirt/libvirtd.conf";
     if (!script_run "ls $libvirtd_conf_file") {
-        script_run "sed -i '/log_level *=/{h;s/^[# ]*log_level *= *[0-9].*\$/log_level = 1/};\${x;/^\$/{s//log_level = 1/;H};x}' $libvirtd_conf_file";
-        script_run "sed -i '/log_outputs *=/{h;s%^[# ]*log_outputs *=.*[0-9].*\$%log_outputs=\"1:file:/var/log/libvirt/libvirtd.log\"%};\${x;/^\$/{s%%log_outputs=\"1:file:/var/log/libvirt/libvirtd.log\"%;H};x}' $libvirtd_conf_file";
+        script_run "sed -i '/log_level *=/{h;s/^[# ]*log_level *= *[0-9].*\$/log_level = 3/};\${x;/^\$/{s//log_level = 3/;H};x}' $libvirtd_conf_file";
```
Comment (guoxuguang) [inline tests/virt_autotest/virt_utils.pm:37]: After basic analyze, we found there was created most 10Gb libvirtd log on sles15sp3 xen host with log_level setting as 1 
Please add the comment here, why changed log_level from 1 to 3 on sles15sp3 xen host without kvm role firstly. 
Then, just only code changed with log_level as 3 for sles15sp3 xen.

Comment (guoxuguang) [reply tests/virt_autotest/virt_utils.pm]: Others, LGTM.

Comment (waynechen55) [reply tests/virt_autotest/virt_utils.pm]: LGTM. Better if tells warning=3 debug=1

Comment (Julie-CAO) [reply tests/virt_autotest/virt_utils.pm]: All done. 

verification run is in progress: https://openqa.nue.suse.com/tests/4706205 & https://openqa.nue.suse.com/tests/4706207

**Comment by alice-suse** [comment]
can it be better to be in 1 commit? others LGTM

**Comment by Julie-CAO** [comment]
> can it be better to be in 1 commit? others LGTM

yes. Done. verification run just passed, https://openqa.nue.suse.com/tests/4706207. Please help merge. Thanks!

**Pull request merged**